kratos/geometries/geometry.h

```diff
@@ -1209,6 +1209,20 @@ class Geometry
         return result;
     }
 
+    /**
+     * @brief It returns a vector that is normal to its corresponding geometry in a given integration method
+     * @param IntegrationPointIndex index of integration point which jacobians has to be calculated in it.
+     * @param rIntegrationPointsArray The integration method which jacobians has to be calculated in its integration points.
```

Comment:
This sentence is wrong. Woud just 'Array containing the integration points corresponding to a particular integration method' work?

Reply:
I will correct those typos